web/src/games/kriegspiel/config.ts

```diff
@@ -0,0 +1,11 @@
+import { IGameConfig } from 'gamesShared/definitions/game';
+import { Kriegspiel } from './Game';
+import { Board } from './Board';
+
+const config: IGameConfig = {
+  bgioGame: Kriegspiel,
+  bgioBoard: Board,
+  debug: true,
```

Comment:
We probably want to leave debug: false before merging.

Reply:
(not resolved)

Reply:
please check the latest version